### PR TITLE
Add entrypoints

### PR DESCRIPTION
**Pull request by MattWellie**
# Fixes

  - the separate scripts that make up the Talos workflow are being referenced by paths to scripts inside the container

## Proposed Changes

  - adds a console_script entrypoint for each separate script
  - I've split out the entrypoint within those scripts to have a cli-accessible main method (where CLI arguments are parsed), and a code-accessible main method (so that I can add a simpler single-step runner for easier use)
  - this PR is almost entirely moving things around, not changing behaviour
  - This will require a PR in prod-pipes

## Checklist

- [ ] Related Issue created
- [ ] Tests covering new change
- [ ] Linting checks pass

### CODE REVIEW

File: talos/hail_filter_sv.py [hunk 1]
```diff
@@ -136,20 +149,30 @@ def main(mt_path: str, panelapp_path: str, pedigree: str, vcf_out: str):
         pedigree ():
         vcf_out (str): where to write VCF out
     """
+    get_logger(__file__).info(
```
Comment (illusional) [inline talos/hail_filter_sv.py:152]: Should define this somewhere central to reuse, given everything is already a script

File: talos/vep_vcf_to_mt.py [hunk 2]
```diff
@@ -28,6 +24,46 @@
     'string': {'type': hl.str, 'default': ''},
     'float': {'type': hl.float64, 'default': 0.0},
 }
+RELEVANT_FIELDS = relevant_fields = [
```
Comment (illusional) [inline talos/vep_vcf_to_mt.py:27]: Is this double assign intentional?

**Review by illusional — COMMENTED** [review]
I quickly scanned, and broadly looks fine - but not explicitly approving because Cas is listed on here, and there are fairly substantial changes outside just CLI + entrypoints (like checkpointing changes)

**Review by cassimons — APPROVED** [review]

**Comment by MattWellie** [comment]
> like checkpointing changes

This was bundled in just to reduce the number of CLI args, a slight behaviour change. We're doing everything in a container and throwing the checkpoint away, so removing the need to pass a specific path. This behaviour might return when we need to support other environments where they might want to recover from a partially-complete run using a persisted checkpoint

File: talos/vep_vcf_to_mt.py (hunk 2 above)
Comment (MattWellie) [reply talos/vep_vcf_to_mt.py]: nope! removed

**Pull request merged**